scripts/src/models/orders.ts

```diff
@@ -337,7 +339,9 @@ class OrderImpl extends CreationDateModel implements Order {
 	@Column("simple-json", { name: "blockchain_data", nullable: true })
 	public blockchainData!: BlockchainData;
 
-	@OneToMany(type => OrderContext, context => context.order)
+	@OneToMany(type => OrderContext, context => context.order, {
+		cascadeInsert: true
```

Comment:
https://github.com/typeorm/typeorm/blob/065e0ceb136d51533e57c0324d271f0bbada70eb/CHANGELOG.md#020:
> replaced cascadeInsert with cascade: ["insert"] syntax from relation options.

Reply:
but that's for typeorm version > `2.0`, and we're (currently) using `0.1.12`